tests/pass-dep/libc/libc-fs.rs

```diff
@@ -17,6 +17,8 @@ mod utils;
 fn main() {
     test_dup();
     test_dup_stdout_stderr();
+    #[cfg(target_os = "macos")]
+    test_ioctl();
```

Comment:
Please add new things at the end unless there's a good reason to put them elsewhere.